java-client/src/main/java/org/apache/pegasus/client/PegasusAdminClient.java

```diff
@@ -230,4 +232,34 @@ public void dropApp(String appName, int reserveSeconds) throws PException {
           String.format("Drop app:%s failed! error: %s.", appName, error.toString()));
     }
   }
+
+  @Override
+  public void listApps(boolean onlyGetAvailableApps, List<app_info> appInfoList) throws PException {
```

Comment:
How about use an enum instead of bool? 

https://betterprogramming.pub/dont-use-boolean-arguments-use-enums-c7cd7ab1876a